### PR TITLE
Feature/docs update

### PR DESCRIPTION
**Pull request by Raphael-create**
Updated screenshots of setups and guides
Updated terminology and changed "coordinator" to "hub"
@bergben @patricklamplmair first adaptations are ready for review

### CODE REVIEW

File: docs/setup-instructions/setup-node-azure-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+![DataSpace-app-settings](img/setup-node-azure-dataspace-app-settings.png)
+
+**6. Virtual machine size**
+
+Here you need to determine the size of the virtual machine on which the DataSpace Node will be running. You can choose between 4 predesigned virtual machine sizes. 
+
+![VM-size](img/setup-node-azure-vm-size.png)
+
+After choosing the size of the virtual machine you can click **Review + create**
```
Comment (patricklamplmair) [inline docs/setup-instructions/setup-node-azure-marketplace.md:37]: Add explanation how the user can access his dataspace node and how I can find the deployment name if he doesn't remember it (managed application parameters).

File: docs/setup-instructions/setup-node-azure-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+**1. Go to Azure Marketplace -** Follow this link: <a href="https://portal.azure.com/#blade/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home/searchQuery/Tributech" target="_blank">Azure Marketplace</a>.
+
+**2.Click on Tributech Dataspace Node** 
+
+Now your screen should look as follows: 
+
+![Startingscreen](img/setup-node-azure-startingscreen.png)
+
+**3. Choose a Hub -** Now you can select between the **DataSpace Node - Trial Hub** and **DataSpace Node - Custom Hub**. After selecting a Hub click on **create**.
```
Comment (patricklamplmair) [inline docs/setup-instructions/setup-node-azure-marketplace.md:11]: Please use a table for a better overview and provide information about public / private plans.

File: docs/setup-instructions/setup-node-azure-marketplace.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Currently you can only choose the Custom Hub.
+
+![Trial-and-custom](img/setup-node-azure-trial-and-custom-hub.png)
+
+**4. Fill in basics** 
+
+First you need to fill out the project details. There you have to choose a subscription. In most cases you will select the **Pay-As-You-Go** subscription.
+You also have to choose a resource group. You can either choose one of your existing resource groups or create a new one.
+Secondly, you need to fill out the instance details. Regarding the region please select **West Europe**. If you look at the manged application name you can see that there is already a previously generated name filled out (dsknode....). The reason for that is that the managed application name needs to be unique. You can also change the name as long as it's unique.
```
Comment (bergben) [inline docs/setup-instructions/setup-node-azure-marketplace.md:20]: *managed (is spelled incorrectly)

**Comment by bergben** [comment]
Good work @Raphael-create. I added a few general notes which are not specific to the goal of this PR but it would be nice if we could include it nonetheless.

**Review by Raphael-create — COMMENTED** [review]
Also capitalized the words: 
- Dataset
- Publication
- Ecosystem
- Request
- Subscription

**Comment by bergben** [comment]
@Raphael-create could you please resolve the comments in this PR from me and @patricklamplmair once you applied the requested changes? 
Judging from the commit messages I think most of it has already been fixed if not everything, thank you!

That we can get a better overview of what must still be done and what is already done so that we can proceed to finalize and mere these changes

**Review by patricklamplmair — APPROVED** [review]

**Pull request merged**